cross/PyNaCl/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME = PyNaCl
+PKG_VERS = 1.3.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://files.pythonhosted.org/packages/source/p/pynacl
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/libsodium
+
+HOMEPAGE = https://github.com/pyca/pynacl/
+COMMENT  = PyNaCl is a Python binding to libsodium, which is a fork of the Networking and Cryptography library.
+LICENSE  = Apache License 2.0
+
+CONFIGURE_TARGET = install-cffi
+
+include ../../mk/spksrc.python-wheel.mk
+
+# libsodium shall be taken from the system libraries
+ENV += SODIUM_INSTALL=system
+
+.PHONY: install-cffi
+install-cffi:
+	@. $(CROSSENV) && build-pip install "cffi==1.14.0"
```

Comment:
I do not understand this requirement. I expect `DEPENDS += cross/cffi` to have same effect - or even better as cross-compilation is concerned

Reply:
Well, actually it doesn't work whenever using crossenv. It needs to be installed in the virtual environment built by crossenv.

Reply:
Let me check again though (I had tried `BUILD_DEPENDS` to be honest)

Reply:
Ok, I rechecked and it doesn't work as `crossenv` actually "forks" a separate environment. (see error raised here: https://github.com/smaarn/spksrc/runs/1205117066?check_suite_focus=true).

To be honest this could simplify lots of cross compiled libraries which currently depend on `cross/pip` or `cross/setuptools` (or event `cross/wheels`) but I'm not too sure about the added values within this PR.

Reply:
ok so I did some changes and basically what I did is to ensure that `cross/cffi` would also install (whenever built within a crossenv compatible way) install itself into cross env setup environments.